wandb/sdk/launch/_project_spec.py

```diff
@@ -127,30 +122,27 @@ def _fetch_project_local(self, api):
             assert utils._GIT_URI_REGEX.match(parsed_uri), (
                 "Non-wandb URI %s should be a Git URI" % parsed_uri
             )
-            utils._fetch_git_repo(parsed_uri, self.git_version, dst_dir)
-        self.dir = dst_dir
-        return self.dir
+            utils._fetch_git_repo(self.dir, parsed_uri, self.git_version)
 
-    def _copy_config_local(self):
+    def _copy_config_local(self) -> None:
         if not self.override_config:
             return None
         if not self.dir:
             dst_dir = tempfile.mkdtemp()
```

Comment:
since the tempdir is now created in init we can remove this -- would prefer fewer possible code paths lying around that do file changes

Reply:
Nice catch!